src/auto_scheduler/measure.cc

```diff
@@ -267,7 +269,11 @@ Array<MeasureResult> ProgramMeasurerNode::Measure(const SearchTask& task,
     }
 
     if (error_ct > max_continuous_error) {
-      LOG(FATAL) << "Too many errors happened during tuning";
+      LOG(WARNING) << "Too many errors happened during tuning. Switching to debug mode."
+                   << std::endl;
+      verbose = 1;
```

Comment:
```suggestion
      verbose = 2;
```
In this PR (https://github.com/apache/incubator-tvm/pull/6882), we changed the verbosity level. Now the debug model is `verboes=2`